ludwig/trainers/trainer.py

```diff
@@ -349,6 +360,34 @@ def write_step_summary(cls, train_summary_writer, combined_loss, all_losses, ste
 
         train_summary_writer.flush()
 
+    def train_for_tuning(
```

Comment:
This function is no longer used. There's a new API called `BatchSizeEvaluator` that we use instead.